packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -896,16 +896,15 @@ describe('plugin-meetings', () => {
           assert.exists(meeting.stopShare);
         });
         beforeEach(() => {
-          meeting.locusInfo.mediaShares = [{name: 'content', url: url1}];
-          meeting.locusInfo.self = {url: url2};
-          meeting.meetingRequest.changeMeetingFloor = sinon.stub().returns(Promise.resolve());
+          meeting.mediaProperties.mediaDirection = {receiveShare: true};
+          meeting.updateShare = sinon.stub().returns(Promise.resolve());
         });
         it('should send the share', async () => {
```

Comment:
Let's update the description for what we are actually testing:

```suggestion
        it('should call updateShare', async () => {
```

Reply:
Thanks @adamweeks  fixed the review comments